wrapping/python/openmeeg/openmeeg.i

```diff
@@ -455,8 +455,12 @@ namespace OpenMEEG {
         dims[0] = ($self)->nlin();
         dims[1] = ($self)->ncol();
 
+        // make a copy of the data
+        double* data = new double[dims[0]*dims[1]];
+        double* start = (*($self)).data();
+        std::copy(start,start+dims[0]*dims[1],data);
```

Comment:
FYI @papadop without this @agramfort (and CIs) were getting segfaults, presumably due to some memory deallocation

Reply:
That's unfortunate... This means that using python will double the amount of memory used... I must say that it has always been difficult to me to handle correctly reference counting with python. One possibility would be to increment OpenMEEG reference, but then we would need to wrap python destructor to decrease the reference counter and I do not know how to do that.

So copying is the best solution in the meantime.... But maybe we can add some comment

Reply:
OK, I just found:
https://numpy.org/doc/stable/reference/c-api/data_memory.html
This seems a way to handle the above problem better.
Instead of copying, we increase the array reference count in OpenMEEG. But we also set a handler whose role would be to:
1) increase the ref counter in case of a shallow copy.
2) decrease the counter when the array is destroyed by python.

I have not looked in depth at the mentioned web page, but that seems a plausible strategy.
Have you added a test that exposed the @agramfort issue (or was it already existing) ???

Reply:
To comment on myself. It's even simpler because we have python ref counting for python clones. We just need to trap the intent of python to free the array and decrease the openmeeg ref counter then...

Reply:
If we don’t do this it makes the current python tests segfault